beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -59,216 +59,218 @@ function LikeButton() {
   const [liked, setLiked] = React.useState(false);
 
   if (liked) {
-    return 'You liked this!';
+    return 'आपको यह पसंद आया!';
```

Comment:
Let's not translate any part of code except the comments

`आपको यह पसंद आया!` => `You liked this`